packages/angular/src/components/popover/popover.ts

```diff
@@ -0,0 +1,181 @@
+import {
+  Component,
+  computed,
+  effect,
+  ElementRef,
+  input,
+  output,
+  signal,
+  viewChild,
+} from '@angular/core'
+import {
+  autoUpdate,
+  computePosition,
+  flip,
+  MiddlewareState,
+  offset,
+  Placement,
+  shift,
+} from '@floating-ui/dom'
+import { Color, SeverityColors } from '../colors'
+import { Size } from '../common-inputs'
+
+@Component({
+  selector: 'ds-popover',
+
+  template: `
+    <div
+      #myPopover
+      popover="manual"
+      class="ds-popover"
+      data-testid="popover"
+      [id]="popoverId()"
+      [attr.data-size]="dataSize()"
+      [attr.data-color]="dataColor()"
+      [attr.data-variant]="variant()"
+    >
+      @if (controlledOpen()) {
+        <ng-content />
+      }
+    </div>
+  `,
+  imports: [],
+})
+export class Popover {
+  // use popoverId instead of id since id will be put on the angular element and not the popover-div
+  readonly popoverId = input.required<string>()
+  readonly placement = input<Placement>('top')
+  readonly autoPlacement = input<boolean>(true)
```

Comment:
Use `transform: booleanAttribute` for boolean-inputs, like this:
`readonly autoPlacement = input(true, { transform: booleanAttribute })`